platform/core/src/utils/roundNumber.js

```diff
@@ -1,5 +1,9 @@
+/**
+ * @param {string | number} value
+ * @param {number} decimals
+ */
 function _round(value, decimals) {
-  return Number(value.toFixed(decimals));
+  return Number(value).toFixed(decimals);
```

Comment:
Please add:  if( typeof value !=='number' ) return;
There are places this function gets passed undefined or null  on load.